apps/ui/lib/controllers/page_controller.ex

```diff
@@ -1,9 +1,7 @@
 defmodule Ui.PageController do
   use Ui, :controller
 
-  require Backend
-
   def index(conn, _params) do
-    render conn, "index.html", quote: Backend.randomQuote 
+    render conn, "index.html", quote: Ui.FortunesClient.fetch_random_quote
```

Comment:
we have FortuneServer and FortunesClient.
I think I mistyped and you followed without thinking. Can we unify please?